packages/ng/callout/src/lib/callout/callout.component.ts

```diff
@@ -0,0 +1,72 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { Palette, SizeMS } from '@lucca-front/ng/core';
+
+@Component({
+	selector: 'lu-callout',
+	standalone: true,
+	imports: [CommonModule],
+	templateUrl: './callout.component.html',
+	styleUrls: ['./callout.component.scss'],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class CalloutComponent {
+	@Input()
+	/**
+	 * The title of the callout
+	 */
+	title: string;
+
+	@Input()
+	/**
+	 * Which palette should be used for the entire callout.
+	 * Defaults to none (inherits parent palette)
+	 */
+	palette: Palette = 'none';
+
+	@Input()
+	/**
+	 * Which size should the callout be? Defaults to medium
+	 */
+	size: SizeMS = 'm';
+
+	@Input()
+	/**
+	 * Should we display the remove icon?
+	 *
+	 * IMPORTANT: the callout won't hide itself, you're responsible for removing
+	 * it using *ngIf as you might want to store the information that it has been
+	 * hidden. Hook on the `hidden` event emitter for that.
+	 */
+	removable = false;
+
+	@Input()
+	/**
+	 * Which icon should we display in the callout if any?
+	 * Defaults to no icon.
+	 */
+	icon: 'info' | 'success' | 'warning' | 'error' | 'help' | string;
```

Comment:
`'any' | 'string' | 'union' | string` is replaced with just `string`. While it is logical, it is sad because we loose auto-completion. Do you prefer to keep it like this for documentation purpose? Or move this list of icons in the comment?

Reply:
As discussed together (but adding it here too for context), this is meant to be a helping hand using suggestions for devs, as the design team allows every icon to be used but recommends these ones. I'll create a helper type for it in a future PR.